core-primitives/top-pool/Cargo.toml

```diff
@@ -60,7 +60,6 @@ byteorder = { version = "1.4.2", default-features = false }
 codec  = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
 derive_more = { version = "0.99.5" }
 log = { version = "0.4", default-features = false }
-retain_mut = { version = "0.1.4"}
```

Comment:
Replaced by the standard library, which now has this included (unstable feature though)